src/main/java/beans/menus/EntityDetailListResponse.java

```diff
@@ -29,12 +27,15 @@ public EntityDetailListResponse(EntityDetailResponse entityDetailResponse) {
         this.isPersistentDetail = true;
     }
 
-    public EntityDetailListResponse(EntityScreen screen, EvaluationContext ec, TreeReference treeReference) {
+    public EntityDetailListResponse(EntityScreen screen, EvaluationContext ec, TreeReference treeReference,
+                                    MenuSession menuSession) {
+        ec.addFunctionHandler(new FormplayerHereFunctionHandler(menuSession, menuSession.getCurrentBrowserLocation()));
```

Comment:
I don't love this pattern of passing the `MenuSession` all the way down the chain to add the function handler here. In all four usages the `EvaluationContext` comes from a call to `menuSession.getSessionWrapper().getEvaluationContext()` - what if we replaced these calls with a call to a new method `menuSession.getEvaluationContext()` that got the EC form the wrapper, then added the `here` handler immediately?